examples/core/gltf/app.js

```diff
@@ -384,6 +384,7 @@ export class DemoApp {
     // TODO: Find better way to do this
     (this.gltf.meshes || []).forEach(mesh => delete mesh._mesh);
     (this.gltf.nodes || []).forEach(node => delete node._node);
+    (this.gltf.bufferViews || []).forEach(bufferView => delete bufferView.lumaBuffers);
```

Comment:
What does this do? Just ensure these get garbage collected? Can't we just set the arrays to empty arrays, that way these objects will no longer be referenced and thus garbage collected.

Reply:
Yes I can do that.